docs/4-functional-programming/homework/src/fp/maybe.ts

```diff
@@ -1,38 +1,44 @@
 export type None = {
-  _tag: 'None'
+  _tag: "None";
 };
 
 export type Some<T> = {
-  readonly _tag: 'Some',
+  readonly _tag: "Some";
   readonly value: T;
-}
+};
 
 /**
  * Maybe is represents an optional value, which might, or might not be.
  * It either a Some<Value>, or None
  */
-export type Maybe<T> = Some<T> | None
+export type Maybe<T> = Some<T> | None;
 
-export const isSome = <T>(optional: Maybe<T>): optional is Some<T> => optional._tag === 'Some';
-export const isNone = <T>(optional: Maybe<T>): optional is None => optional._tag === 'None';
+export const isSome = <T>(optional: Maybe<T>): optional is Some<T> =>
+  optional._tag === "Some";
+export const isNone = <T>(optional: Maybe<T>): optional is None =>
+  optional._tag === "None";
 
 export const some = <T>(value: T): Some<T> => ({
-  _tag: 'Some',
+  _tag: "Some",
   value,
 });
 export const none: Readonly<None> = {
-  _tag: 'None',
+  _tag: "None",
 };
 
 /**
  * Create a Maybe instance form the value. If value(T) is nullable(null or undefined), returns None, otherwise it returns Some<T>
  */
-export const fromNullable = <T>(value: T) => ();
+export const fromNullable = <T>(value: T | null | undefined): Maybe<T> =>
+  value === null || value === undefined ? none : some(value);
```

Comment:
I think we can combine this statement as

from 
```
value === null || value === undefined ? none : some(value);
```

to 
```
value ? none : some(value);
```

Reply:
Surely,  I updated it!